src/terminal-search-dialog.c

```diff
@@ -350,13 +353,15 @@ terminal_search_dialog_get_regex (GtkWidget *dialog)
 
 	pattern = text = terminal_search_dialog_get_search_text (dialog);
 
-	compile_flags = G_REGEX_OPTIMIZE;
+	// What is the equivalent for this in PCRE?
+	// compile_flags = G_REGEX_OPTIMIZE;
```

Comment:
I don't know, this commit help you? https://github.com/GNOME/gnome-terminal/commit/ef7a345fedd40a69836dfb9c226983e95f0d2e1f

Reply:
Thanks, updated.